scripts/custom/clean_osiris_codes.py

```diff
@@ -0,0 +1,9 @@
+# function clean_osiris_codes
+# remove all chars before and including the first colon
+
+
+def clean_osiris_codes(raw_input):
+    if raw_input is None:
+        return None
+    firstColonIndex = raw_input.find(":") + 1
+    return raw_input[firstColonIndex:]
```

Comment:
What you could also do here for a more general cleaning script is 
-write a dict with codes of all terminology services accepted by fhir, 
-use a regex to see if it is part of the raw_input (you can use re.search() in python for that purpose) 
- if it is, return the code associated